.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "pac-cmd"]
```

Comment:
pac has no runtime dependencies on pac-cmd. We can add to flashlight-build when integration.